cli/man/splinter-upgrade.1.md

```diff
@@ -0,0 +1,49 @@
+% SPLINTER-UPGRADE(1) Cargill, Incorporated | Splinter Commands
+<!--
+  Copyright 2018-2021 Cargill Incorporated
+  Licensed under Creative Commons Attribution 4.0 International License
+  https://creativecommons.org/licenses/by/4.0/
+-->
+
+NAME
+====
+
+**splinter-upgrade** — Upgrades splinter yaml state to database state
+
+SYNOPSIS
+========
+
+**splinter** **upgrade** \[**FLAGS**\]
+
+DESCRIPTION
+===========
+Upgrades splinter by importing data from the deprecated yaml state file format
```

Comment:
Can you add some information here about how the files are renamed? Also add an EXAMPLE and SEE ALSO section

Reply:
Updated!